detection-rules/body_html_smuggling_atob_in_body.yml

```diff
@@ -0,0 +1,13 @@
+name: "Body HTML: HTML smuggling with atob in message body"
+description: | 
+  Detects if the email body HTML contains the document write or insertAdjacentHTML method and atob function call. 
+  This technique has been observed leading to credential phishing. 
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and (length(body.plain.raw) < 200 or body.plain.raw is null)
+  and strings.ilike(body.html.raw, "*document*write*atob*", "*document*insertAdjacentHTML*atob*")
```

Comment:
this will match _either_ one of these strings, doesn't have to be both -- just making sure that's intended